packages/ui/src/atom/label-tooltip-atom/label-tooltip-atom.theme-default.scss

```diff
@@ -0,0 +1,15 @@
+.label-tooltip-atom.theme-default {
+  display: inline-block;
+  vertical-align: middle;
+}
+.label-tooltip-atom.theme-default .icon-atom {
```

Comment:
Component should no know something about children. Create new component and control layout behavior through icon wrapper.

Reply:
just passed classname to icon-atom